### ISSUE TITLE
please add this feature

### ISSUE THREAD

**Issue by DipaliN97** [creative-tutorials/chatbot#1]
Does not vanish the previous messages of current chat and the chatbot message should be comes in the below of human message. and also add one button type message also.

thanks and regards,
Dipali Narkhede 
CC: @creative-tutorials

**Comment by creative-tutorials** [comment]
What do you mean by button message

**Comment by DipaliN97** [comment]
> What do you mean by button message

it means for example if user send a message like career options then bot must send the three buttons like education ,cricket, entertainment so user can make a choice by clicking on button.

**Comment by creative-tutorials** [comment]
> > What do you mean by button message
> 
> it means for example if user send a message like career options then bot must send the three buttons like education ,cricket, entertainment so user can make a choice by clicking on button.

Okay I'll work on it, but it might take a while I'm very busy at work, so I usually don't have time for YouTube again, but give me enough time I'll surely update & push changes

**Comment by DipaliN97** [comment]
> > > What do you mean by button message
> > 
> > 
> > it means for example if user send a message like career options then bot must send the three buttons like education ,cricket, entertainment so user can make a choice by clicking on button.
> 
> Okay I'll work on it, but it might take a while I'm very busy at work, so I usually don't have time for YouTube again, but give me enough time I'll surely update & push changes

okay actually I am working on it but I fail to do these things please do this as quick as you possible from your busy  schedule.

**Comment by creative-tutorials** [comment]
> > > > What do you mean by button message
> > > 
> > > 
> > > it means for example if user send a message like career options then bot must send the three buttons like education ,cricket, entertainment so user can make a choice by clicking on button.
> > 
> > 
> > Okay I'll work on it, but it might take a while I'm very busy at work, so I usually don't have time for YouTube again, but give me enough time I'll surely update & push changes
> 
> okay actually I am working on it but I fail to do these things please do this as quick as you possible from your busy schedule.

Okay, are you on Discord we can do it together 😎

**Comment by DipaliN97** [comment]
> > > > > What do you mean by button message
> > > > 
> > > > 
> > > > it means for example if user send a message like career options then bot must send the three buttons like education ,cricket, entertainment so user can make a choice by clicking on button.
> > > 
> > > 
> > > Okay I'll work on it, but it might take a while I'm very busy at work, so I usually don't have time for YouTube again, but give me enough time I'll surely update & push changes
> > 
> > 
> > okay actually I am working on it but I fail to do these things please do this as quick as you possible from your busy schedule.
> 
> Okay, are you on Discord we can do it together 😎

yes sure we can do together. tell me when you are free.
or I will share a repository on my account what I did so you can make it a changes in it.

**Comment by creative-tutorials** [comment]
> 

This seems better. Okay my discord username is Timi#7396, We'll start tomorrow

**Comment by creative-tutorials** [comment]
> > > > > > What do you mean by button message
> > > > > 
> > > > > 
> > > > > it means for example if user send a message like career options then bot must send the three buttons like education ,cricket, entertainment so user can make a choice by clicking on button.
> > > > 
> > > > 
> > > > Okay I'll work on it, but it might take a while I'm very busy at work, so I usually don't have time for YouTube again, but give me enough time I'll surely update & push changes
> > > 
> > > 
> > > okay actually I am working on it but I fail to do these things please do this as quick as you possible from your busy schedule.
> > 
> > 
> > Okay, are you on Discord we can do it together 😎
> 
> yes sure we can do together. tell me when you are free. or I will share a repository on my account what I did so you can make it a changes in it.

This seems better. Okay my discord username is Timi#7396, We'll start tomorrow

**Comment by DipaliN97** [comment]
> > > > > > What do you mean by button message
> > > > > 
> > > > > 
> > > > > it means for example if user send a message like career options then bot must send the three buttons like education ,cricket, entertainment so user can make a choice by clicking on button.
> > > > 
> > > > 
> > > > Okay I'll work on it, but it might take a while I'm very busy at work, so I usually don't have time for YouTube again, but give me enough time I'll surely update & push changes
> > > 
> > > 
> > > okay actually I am working on it but I fail to do these things please do this as quick as you possible from your busy schedule.
> > 
> > 
> > Okay, are you on Discord we can do it together 😎
> 
> yes sure we can do together. tell me when you are free. or I will share a repository on my account what I did so you can make it a changes in it.


> > > > > > > What do you mean by button message
> > > > > > 
> > > > > > 
> > > > > > it means for example if user send a message like career options then bot must send the three buttons like education ,cricket, entertainment so user can make a choice by clicking on button.
> > > > > 
> > > > > 
> > > > > Okay I'll work on it, but it might take a while I'm very busy at work, so I usually don't have time for YouTube again, but give me enough time I'll surely update & push changes
> > > > 
> > > > 
> > > > okay actually I am working on it but I fail to do these things please do this as quick as you possible from your busy schedule.
> > > 
> > > 
> > > Okay, are you on Discord we can do it together 😎
> > 
> > 
> > yes sure we can do together. tell me when you are free. or I will share a repository on my account what I did so you can make it a changes in it.
> 
> This seems better. Okay my discord username is Timi#7396, We'll start tomorrow

okay when we can connect today. tell me timing when you are free.

**Comment by DipaliN97** [comment]
Dipali97#4950 this is my username

**Comment by DipaliN97** [comment]
> > > > > > > What do you mean by button message
> > > > > > 
> > > > > > 
> > > > > > it means for example if user send a message like career options then bot must send the three buttons like education ,cricket, entertainment so user can make a choice by clicking on button.
> > > > > 
> > > > > 
> > > > > Okay I'll work on it, but it might take a while I'm very busy at work, so I usually don't have time for YouTube again, but give me enough time I'll surely update & push changes
> > > > 
> > > > 
> > > > okay actually I am working on it but I fail to do these things please do this as quick as you possible from your busy schedule.
> > > 
> > > 
> > > Okay, are you on Discord we can do it together 😎
> > 
> > 
> > yes sure we can do together. tell me when you are free. or I will share a repository on my account what I did so you can make it a changes in it.
> 
> > > > > > > > What do you mean by button message
> > > > > > > 
> > > > > > > 
> > > > > > > it means for example if user send a message like career options then bot must send the three buttons like education ,cricket, entertainment so user can make a choice by clicking on button.
> > > > > > 
> > > > > > 
> > > > > > Okay I'll work on it, but it might take a while I'm very busy at work, so I usually don't have time for YouTube again, but give me enough time I'll surely update & push changes
> > > > > 
> > > > > 
> > > > > okay actually I am working on it but I fail to do these things please do this as quick as you possible from your busy schedule.
> > > > 
> > > > 
> > > > Okay, are you on Discord we can do it together 😎
> > > 
> > > 
> > > yes sure we can do together. tell me when you are free. or I will share a repository on my account what I did so you can make it a changes in it.
> > 
> > 
> > This seems better. Okay my discord username is Timi#7396, We'll start tomorrow
> 
> okay when we can connect today. tell me timing when you are free.
I think our timing will not be matching so when you are free please you can do. i uploaded app.js and app.css file in a repository on my profile  whichever changes i did.

**Comment by DipaliN97** [comment]
> > > > > > > > What do you mean by button message
> > > > > > > 
> > > > > > > 
> > > > > > > it means for example if user send a message like career options then bot must send the three buttons like education ,cricket, entertainment so user can make a choice by clicking on button.
> > > > > > 
> > > > > > 
> > > > > > Okay I'll work on it, but it might take a while I'm very busy at work, so I usually don't have time for YouTube again, but give me enough time I'll surely update & push changes
> > > > > 
> > > > > 
> > > > > okay actually I am working on it but I fail to do these things please do this as quick as you possible from your busy schedule.
> > > > 
> > > > 
> > > > Okay, are you on Discord we can do it together sunglasses
> > > 
> > > 
> > > yes sure we can do together. tell me when you are free. or I will share a repository on my account what I did so you can make it a changes in it.
> > 
> > 
> > > > > > > > > What do you mean by button message
> > > > > > > > 
> > > > > > > > 
> > > > > > > > it means for example if user send a message like career options then bot must send the three buttons like education ,cricket, entertainment so user can make a choice by clicking on button.
> > > > > > > 
> > > > > > > 
> > > > > > > Okay I'll work on it, but it might take a while I'm very busy at work, so I usually don't have time for YouTube again, but give me enough time I'll surely update & push changes
> > > > > > 
> > > > > > 
> > > > > > okay actually I am working on it but I fail to do these things please do this as quick as you possible from your busy schedule.
> > > > > 
> > > > > 
> > > > > Okay, are you on Discord we can do it together sunglasses
> > > > 
> > > > 
> > > > yes sure we can do together. tell me when you are free. or I will share a repository on my account what I did so you can make it a changes in it.
> > > 
> > > 
> > > This seems better. Okay my discord username is Timi#7396, We'll start tomorrow
> > 
> > 
> > okay when we can connect today. tell me timing when you are free.
> > I think our timing will not be matching so when you are free please you can do. i uploaded app.js and app.css file in a repository on my profile  whichever changes i did.

can you please try to help me?

**Comment by DipaliN97** [comment]
hey are you their is you have time tomorrow?

**Comment by creative-tutorials** [comment]
> 

Yes I have sent you a request on Discrord

**Comment by creative-tutorials** [comment]
> hey are you their is you have time tomorrow?

Yes I have sent you a friend request on Discord

**Comment by DipaliN97** [comment]
> > hey are you their is you have time tomorrow?
> 
> Yes I have sent you a friend request on Discord
yes I accepted tell me the timing so i will be available at that time.
